docs/explanations/audit-scanner/limitations.md

```diff
@@ -51,7 +52,10 @@ spec:
         - UPDATE
 ```
 
-The `apiGroups`, `apiVersions` and `resources` attributes can use the special `*` inside of them. This is a wildcard
-symbol that causes the policy to match all the values used inside of the field.
+<!--TODO:Is this OK, to me it looks like we are saying how to use a
+wildcard. Then we say that wildcards are ignored. Is that right?-->
```

Comment:
We are pointing out that it's fine to use wildcards for certain fields of a policy CR. However, the user must be aware that these type of policies are not going to be evaluated by the audit checks.